src/core/fonts.js

```diff
@@ -2647,6 +2647,8 @@ var Font = (function FontClosure() {
         // view of the sanitizer
         data = this.checkAndRepair(name, file, properties);
         if (this.isOpenType) {
+          adjustWidths(properties);
```

Comment:
We *might* end up parsing the font as CFF, but we could also not enter https://github.com/Snuffleupagus/pdf.js/blob/adjustWidths/src/core/fonts.js#L4078 and not use CFF in the end. I'm not sure what will happen if we call `adjustWidths` for that case too, so it might be more safe to only call this if we know for sure that CFF parsing has been done.